tests/src/org.hibernate.orm/hibernate-core/7.2.0.Final/generate-metadata.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright and related rights waived via CC0
+#
+# You should have received a copy of the CC0 legalcode along with this
+# work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+
+../../../../../gradlew deleteGeneratedMetadata
```

Comment:
@kimeta how would this work with your latest automatic fixes? Maybe we need to augment our scripts to catch also `generate-metadata.sh`?

We should add this then into the `CONTRIBUTING.md` guide.